### ISSUE TITLE
What are the user's solutions of Focus Not Obscured (minimum)?

### ISSUE THREAD

**Issue by YusukeHirao** [w3c/wcag#2700]
I understand that a user can succeed if the focused element shows depending on the scroll position that the user moved on purpose. Because it says:

> Sufficient Techniques
> CSS: Using scroll-padding to ensure a sticky header does not obscure the focused item (Potential future technique).

And I also understand that it is a success if the overwrapped element is hideable. Ref: https://github.com/w3c/wcag/pull/2611

However, what about other user actions? For example, a user zooms it,  resizes the view, and rotates the device. I guess those are not solutions because there are devices, people, and environments that can and can't. 

In short, are the user's solutions only clicking or scrolling realistically?

I understand the SC should be abstract. And it probably is difficult that list those.

I leave this question for developers with the same interrogative in the future.

**Comment by detlevhfischer** [comment]
@YusukeHirao  Not sure if you mean end users or authors. 

The only solution for **end users** in situations where the focus is obscured by fixed positioned headers or footers is, for footers, tabbing to advance the scroll position and then tab backwards (or vice versa for fixed headers). 

What **authors** can do is:
* avoid fixed content altogether (especially fixed footers); 
* make sure controls in fixed footers (such as cookie banners) receive focus at the start of the page's source code so users can close them before tabbing through content
* Use techniques such as scroll-padding to ensure the keyboard focus is visible despite the exisitence of fixed content areas.
* Conceivably, offer options or settings to 'unstick' such content (this is likely not an ideal solution since such options would have to be identified, found, understood)

Does that answer your question? If so, please close the issue.

**Comment by alastc** [comment]
I guess the question is more about what happens if you zoom in and the wrapping changes so that the focused element is not hidden.

I overlaps with the concept of testing page-variations, so I think the answer is that this SC should be tested across page variations.

**Comment by YusukeHirao** [comment]
Sorry for the late confirmation and my replying.
( @MakotoUeki Thanks for reminding me. )

> I overlaps with the concept of testing page-variations, so I think the answer is that this SC should be tested across page variations.

This means that focus cannot be hidden in any way, such as by zooming, resizing the view, or rotating the device.

Do I understand correctly?

**Comment by alastc** [comment]
> This means that focus cannot be hidden in any way, such as by zooming, resizing the view, or rotating the device.

If the user zooms, resizes or rotates the device that may hide the focused item, that is ok (not a failure in itself). However, the next press of `tab` should bring the item into the viewport.

The requirement for authors is to ensure focused items do not get hidden by authored content, including page variations (e.g. where you get to _after_ zooming / page-sizing / rotation).

E.g. if a footer re-sizes as you zoom-in and obscures the currently focused item, that does not fail. If that item is hidden as you tab to it at that zoom level, that would fail.

Detlev's answer above was accepted by the group, so closing this issue. Please re-open if you think something is not answered /resolved.
